samples/variants/build.gradle

```diff
@@ -1,6 +1,6 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
-//apply plugin: 'com.datadoghq.dd-sdk-android-gradle-plugin'
```

Comment:
it seems this change should be reverted, this is a valid id per https://github.com/DataDog/dd-sdk-android-gradle-plugin/blob/0c19f89d19fbbe1de91cd0580b0f2b9fc3cdbf1b/dd-sdk-android-gradle-plugin/build.gradle.kts#L70